MetaMask/metamask-extension
# UX Multichain: Added background color of test networks

This PR is to add the background color of the test network that matches the with network menu list in PickerNetwork (app-header) and  TokenListItem.

* Fixes #20001 

## Screenshots/Screencaps
### Before

![Screenshot 2023-07-14 at 8 50 33 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/904212dc-ff7b-497a-a3a3-6c8f816d72e0)

### After

https://github.com/MetaMask/metamask-extension/assets/39872794/4f49646c-530a-4543-aa00-35e71ece102e



## Manual Testing Steps

- Go to test network
- Check the network icon has same background color in token-list badge and network menu

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
